### PR TITLE
Clarify WIP PR instructions

### PR DESCRIPTION
**Pull request by Mr0grog**
A new contributor noted [some confusion from the text around WIP PRs](https://github.com/edgi-govdata-archiving/web-monitoring-ui/pull/424), so this attempts to clean up the text around that part of our contributor guide. It also switches from recommending “WIP” in the title to using GitHub’s “draft” feature (didn’t exist when this guide was written!) and notes that you should put issue numbers in commit messages.

### CODE REVIEW

**Review by Frijol — APPROVED** [review]
Whoa I had no idea about the draft PR feature! That's great.

File: CONTRIBUTING.md
Comment (lightandluck) [inline CONTRIBUTING.md]: This was already in the document, but reading the sentence here, it's hard to understand with all the parenthetical's.  Suggest we update
```suggestion
1. Once your changes are ready for review, push your commits to GitHub and **[create a pull request (PR)](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).** Assign as a reviewer or ping a Lieutenant active on the project. Lieutenants are able to merge in PRs. They can be pinged individually using "`@<username>`" or all Lieutenants can be pinged via `@edgi-govdata-archiving/lieutenants`.
```

**Review by lightandluck — APPROVED** [review]
Looks great! Left one suggestion for updating some old language. Approving anyway because getting ready to fly. Probably won't be able to look at again today and don't want to hold up this PR.

File: CONTRIBUTING.md
Comment (Mr0grog) [reply CONTRIBUTING.md]: Yeah, I thought this wording was a bit awkward, too, but was trying to be relatively minimal.

What if we just drop this part? I don’t think a non-member of a repo can assign reviewers, and I don’t think there’s much gained by having them try to figure out who to actually @-mention. People only do that approximately 1/100th of the time anyway, and I’m not sure it improves my workflow as the repo maintainer/person triaging issues & PRs. (Does having this in the initial PR post help you?)

> 1. Once your changes are ready for review, push your commits to GitHub and **[create a pull request (PR)](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).** If you aren’t ready for final review and just need some preliminary feedback, create the PR as a *draft:*

Comment (Mr0grog) [reply CONTRIBUTING.md]: I also don’t think I’ve ever seen anyone @ the whole lieutenants group, and not sure it would be useful except as a last resort for a PR that’s been sitting with no feedback for weeks or something. (Most people in that group are not super active these days, and in general only one or two of the 20 of them would be a person relevant to any given repo.)

Comment (lightandluck) [reply CONTRIBUTING.md]: Yea, I've never seen anyone use the Lieutenant thing either and few are active in that group. Let's axe it. 

The draft option info helps!

Comment (Mr0grog) [reply CONTRIBUTING.md]: DONE.

**Pull request merged**
